### ISSUE TITLE
Images served as text/html under Tomcat 9.0.16

### ISSUE THREAD

**Issue by ilgrosso** [gitbucket/gitbucket#2282]
## Issue
**Impacted version**: 4.30.1

**Deployment mode**: Tomcat 9.0.16, with MySQL

**Problem description**:

All image are served with content-type `text/html`; as a result, the Gitbucket logo on the top left corner (SVG) is not displayed.

```
$ curl -O -v http://quercia:8080/gitbucket/assets/common/images/gitbucket.svg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0*   Trying 192.168.0.254...
* TCP_NODELAY set
* Connected to quercia (192.168.0.254) port 8080 (#0)
> GET /gitbucket/assets/common/images/gitbucket.svg HTTP/1.1
> Host: quercia:8080
> User-Agent: curl/7.58.0
> Accept: */*
> 
< HTTP/1.1 200 
< Set-Cookie: JSESSIONID=E50E1AA46059E099D939E822290F79D5; Path=/gitbucket; Secure; HttpOnly
< Accept-Ranges: bytes
< ETag: W/"2262-1545443382000"
< Last-Modified: Sat, 22 Dec 2018 01:49:42 GMT
< Content-Type: text/html;charset=UTF-8
< Content-Length: 2262
< Date: Fri, 15 Mar 2019 09:08:39 GMT
< 
{ [2262 bytes data]
100  2262  100  2262    0     0  59526      0 --:--:-- --:--:-- --:--:-- 59526
* Connection #0 to host quercia left intact
```

Images from other applications in the same Tomcat instance are served correctly, instead:

```
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0*   Trying 192.168.0.254...
* TCP_NODELAY set
* Connected to quercia (192.168.0.254) port 8080 (#0)
> GET /tomcat.png HTTP/1.1
> Host: quercia:8080
> User-Agent: curl/7.58.0
> Accept: */*
> 
< HTTP/1.1 200 
< Accept-Ranges: bytes
< ETag: W/"5103-1549297867000"
< Last-Modified: Mon, 04 Feb 2019 16:31:07 GMT
< Content-Type: image/png
< Content-Length: 5103
< Date: Fri, 15 Mar 2019 09:10:32 GMT
< 
{ [2697 bytes data]
100  5103  100  5103    0     0   415k      0 --:--:-- --:--:-- --:--:--  415k
* Connection #0 to host quercia left intact
```

**Comment by kounoike** [comment]
Thanks for reporting, but already reported in #2218. and fixed in #2241. Please use 4.31.1.

**Comment by ilgrosso** [comment]
Ouch It seems I overlooked that, and was also convinced to run the latest version...
